frontend/src/metabase/search/components/SearchResult.unit.spec.js

```diff
@@ -23,6 +25,50 @@ function collection({
   return collection;
 }
 
+describe("SearchResult > Tables", () => {
+  it("tables with initial_sync_status='incomplete' are not clickable", async () => {
+    const table = { id: 1, display_name: "Table Name" };
+    const database = { id: 1, name: "Database Name" };
+    fetchMock.get("path:/api/table/1", table);
```

Comment:
We have [`setupTableEndpoints`](https://github.com/metabase/metabase/blob/38c95b047beddb1c9da00f9d845e1d00b8d41c1e/frontend/test/__support__/server-mocks/table.ts#L5), maybe we could use that?